cep-sigstore-predicate.md

```diff
@@ -0,0 +1,366 @@
+# CEP - Standardizing a publish attestation for the conda ecosystem
+
+<table>
+<tr><td> Title </td><td> Standardizing a publish attestation for the conda ecosystem </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Wolf Vollprecht &lt;wolf@prefix.dev&gt;, William Woodruff &lt;william.woodruff@trailofbits.com&gt;</td></tr>
+<tr><td> Created </td><td> Feb 18, 2025 </td></tr>
+<tr><td> Updated </td><td> Jun 18, 2025</td></tr>
+<tr><td> Discussion </td><td>  </td></tr>
+<tr><td> Implementation </td><td>  </td></tr>
+</table>
+
+## Abstract
+
+This CEP proposes a standard attestation layout for the conda ecosystem.
+This attestation layout is based on the [in-toto] framework
+and will enable further integration with signing schemes like
+[Sigstore].
+
```

Comment:
As a stylistic note, we should add something like this, since it looks like we're using "MUST" in the RFC2119-way:

> ## Specification
>
> The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][RFC2119] when, and only when, they appear in all capitals, as shown here.
>
> More specifically, violations of a MUST or MUST NOT rule MUST result in an error. Violations of the rules specified by any of the other all-capital terms MAY result in a warning, at discretion of the implementation.